spec/policy/keycloak_role_check/keycloak_role_check_spec.lua

```diff
@@ -0,0 +1,572 @@
+local KeycloakRoleCheckPolicy = require('apicast.policy.keycloak_role_check')
+
+describe('Keycloak Role check policy', function()
+
+  before_each(function()
+    ngx.header = {}
+  end)
+
+  describe('.access', function()
```

Comment:
I think we should try to write `describe` and `it` comments in a way that they form a correct sentence. In my opinion, that would make the specs easier to follow and it would express more clearly the expected behavior of the policy.

Take the specs for the CORS policy for example:
https://github.com/3scale/apicast/blob/a95ebddd2bc25e953fb8096ce181cd5123ec20dd/spec/policy/cors/cors_spec.lua

For the specs of the `.rewrite` function, if we read the `describe` and `it` clauses together we get:
1) when the request is a CORS preflight, it exits with status code 204.
2) when the request is not a CORS preflight, it does not exit.

I don't think this should be a blocker for merging this PR. However, I think this is something we should take into account for future PRs.

Reply:
Improved.